tests/data/test_policies/cvm.yaml

```diff
@@ -0,0 +1,8 @@
+policies:
```

Comment:
this should move to tools/c7n_tencentcloud/tests/data/policies

Reply:
Done
commit: [e7ff155](https://github.com/cloud-custodian/cloud-custodian/pull/7688/commits/e7ff1551bc93a998fb3b23d3808042eaac9d976d)